src/utils/mixins.js

```diff
@@ -183,60 +186,64 @@ const fileUploader = {
       postUpload = async () => {},
       uploadCls = Upload,
     }) {
-      const results = [];
+      const promiseList = [];
       this.uploading = true;
+      this.indeterminate = true;
       this.errorMessage = null;
       await preUpload();
-      for (; this.currentIndex < this.files.length; this.currentIndex += 1) {
-        const file = this.files[this.currentIndex];
+      this.indeterminate = false;
+      for (let i = 0; i < this.files.length; i += 1) {
```

Comment:
This looks like it could be replaced with `const promiseList = this.files.map(...)`

Reply:
I'm pretty sure `const promise list` is the name of a love song.